client/src/components/app/Field.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types";
```

Comment:
it is better to create a `ui` directory that will contain all customized ui elements.
this can be placed in `client/src/components/ui/`

Reply:
Done